src/main/scala/md/fusionworks/adam/jbrowse/models/TrackList.scala

```diff
@@ -75,4 +226,57 @@ case class TracksConf(
                      key: String
                        )
 
+case class Global(
+                   featureDensity: Double,
+                   featureCount: Int,
+                   scoreMin: Int,
+                   scoreMax: Int,
+                   scoreMean: Int,
+                   scoreStdDev: Double
+                   )
+
+case class Features(
+                     features: List[Feature]
+                     )
+
+
+case class Feature(
+                    start:Int,
+                    end: Int
+                    )
+
+case class Feature2(
```

Comment:
I am very excited with you case class naming convention! The most powerful part of it is that it has no limits since integer numbers are not limited in size when written as string.